src/tools/run_manifest_validation/index.ts

```diff
@@ -0,0 +1,35 @@
+import runValidation from "./runValidation.js";
+import {inputSchema, outputSchema} from "./schema.js";
+import {getLogger} from "@ui5/logger";
+import Context from "../../Context.js";
+import {RegisterTool} from "../../registerTools.js";
+
+const log = getLogger("tools:run_manifest_validation");
+
+export default function registerTool(registerTool: RegisterTool, _context: Context) {
+	registerTool("run_manifest_validation", {
+		title: "Manifest Validation",
+		description:
+			"Validates UI5 manifest file." +
+			"After making changes, you should always run the validation again " +
+			"to verify that no new problems have been introduced.",
+		annotations: {
+			title: "Manifest Validation",
+			readOnlyHint: false,
+		},
+		inputSchema,
+		outputSchema,
+	}, async ({manifestPath}) => {
+		log.info(`Running manifest validation on ${manifestPath}...`);
+
+		const result = await runValidation(manifestPath);
```

Comment:
Just like for the create tool, please normalize the path using the Context:

https://github.com/UI5/mcp-server/blob/f5035db83d6838c76d57cdb32ce62089b8184b89/src/tools/create_integration_card/index.ts#L27